pandas/core/indexes/base.py

```diff
@@ -2622,7 +2636,7 @@ def get_indexer(self, target, method=None, limit=None, tolerance=None):
 
     def _convert_tolerance(self, tolerance):
         # override this method on subclasses
-        return tolerance
```

Comment:
instead of adding this function, just do

```
def _convert_tolerance(self, tolerance):
    if is_scalar(tolerance):
       return tolernace
    return np.asarray(tolerance)
```

you *may* be able to get away with just ``np.asarray(tolerance)`` (as ndarray indexing knows how to handle scalar things)

Reply:
eg. this